### DISCUSSION TITLE
Web UI keeps on failing This site can’t be reached took too long to respond.

### DISCUSSION THREAD

**Discussion by mjombanorman** [Security-Onion-Solutions/securityonion#11628] [2.4]
### Version

2.4.2

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

32

### Storage for /

263

### Storage for /nsm

728

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I've noticed that I must restart the Firewalld for Web UI access every time I need to access it. System checks show no issues, but the Web UI remains inaccessible. Any solutions or tips for this?

![Screenshot 2023-10-26 160455](https://github.com/Security-Onion-Solutions/securityonion/assets/9483695/5c4eed2c-c227-4907-9fde-18c7b5a1b840)
![Screenshot 2023-10-26 160643](https://github.com/Security-Onion-Solutions/securityonion/assets/9483695/c3eed284-566b-4159-9cd7-01f259c7fc8d)
### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by jaypfe** [comment]
Perhaps the devices IP you are using to access it changed to something that is no longer allowed through the firewall. 
In the setup, you allowed the analyst IP of the device you are using?

**Reply by mjombanorman** [reply]
I have checked my firewall requests are going through and I am able to ssh into the server via the same subnet which I had while listed.

**Reply by jaypfe** [reply]
SSH and web UI are different. 

Try running this command on the manager:

sudo so-firewall includehost analyst <YOUR IP ADDRESS>

**Reply by mjombanorman** [reply]
This was the first task post-installation.I can see the IP whitelisted in the iptables -L -v command.

**Comment by jaypfe** [comment]
You see it allowed for both http and https?
